double-conversion/bignum.cc

```diff
@@ -611,10 +624,10 @@ bool Bignum::ToHexString(char* buffer, int buffer_size) const {
 }
 
 
-Bignum::Chunk Bignum::BigitAt(int index) const {
+Bignum::Chunk Bignum::BigitOrZero(int index) const {
```

Comment:
Good change.

Reply:
Thanks. In the spirit of "the least surprise"...